ModelFiles/yml/yeastGEM.yml

```diff
@@ -93995,6 +93996,16 @@
     - annotation: !!omap
       - sbo: SBO:0000395
     - confidence_score: 0
+  - !!omap
+    - id: r_4600
+    - name: Ca(2+) exchange
+    - metabolites: !!omap
+      - s_4199: -1
+    - lower_bound: -1000
+    - upper_bound: 1000
+    - annotation: !!omap
+      - sbo: SBO:0000627
+    - confidence_score: 1
```

Comment:
My only comment: Note that for exchange rxns, confidence scores should be = `NaN`, based on our [definitions](https://github.com/SysBioChalmers/yeast-GEM/blob/master/ComplementaryScripts/missingFields/addConfidenceScores.m). Other than that looks great, thanks for realizing we were neglecting Calcium :)